src/inputs/input.scss

```diff
@@ -1,27 +1,26 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
-@import './variables';
 
 @mixin iui-input {
```

Comment:
I opened this PR expecting to find 2-3 lines of code added for settings small and large sizes. But it seems you're changing a bunch of different things.

And yet it does not fix the two real world problems we have encountered multiple times 😄 
1. inline inputs have issues (with icon, with message, and with required label)
2. readonly inputs look like disabled and are almost unusable in that state

Reply:
I can handle the readonly inputs in this PR, no problem.
![Screen Shot 2021-05-24 at 12 18 00 PM](https://user-images.githubusercontent.com/849817/119376560-f18f6e00-bc70-11eb-9689-5452ebe44218.png)

I'd like to tackle the inline inputs icon / message issue separately.